Makefile

```diff
@@ -40,84 +40,57 @@ cri_version   =  release-1.26
 # Configuration Options
 export GIT_PAGER = cat
 
+#------------------------------------------------------------------------------#
+
+# Convenience commands
 
 .PHONY: all
 all: install ## Build and install (debug) 🎉
 
-.PHONY: install
-install: build ## Build and install (debug) 🎉
-
 .PHONY: build
-build: musl auraed auraescript ## Build and install (debug) (+musl) 🎉
+build: install ## Build and install (debug) (+musl) 🎉
 
-.PHONY: prcheck
-prcheck: build lint test-all ## Meant to mimic the GHA checks (includes ignored tests)
+#------------------------------------------------------------------------------#
 
-.PHONY: lint
-lint: ## Lint the code
-	@$(cargo) clippy -p auraed --target $(uname_m)-unknown-linux-musl --all-features -- -D clippy::all -D warnings
-	@$(cargo) clippy -p auraescript --all-features -- -D clippy::all -D warnings
-	@$(cargo) clippy -p aer --all-features -- -D clippy::all -D warnings
-	@$(cargo) clippy -p aurae-client --all-features -- -D clippy::all -D warnings
+# Super commands
 
-.PHONY: release
-release: proto## Build (static+musl) and install (release) 🎉
-	$(cargo) install --target $(uname_m)-unknown-linux-musl --path ./auraed
-	$(cargo) install --path ./auraescript
+.PHONY: clean
+clean: clean-certs clean-gens clean-crates ## Clean the repo
 
-.PHONY: auraescript
-auraescript: proto ## Initialize and compile auraescript
-	@$(cargo) clippy -p auraescript
-	@$(cargo) install --path ./auraescript --debug --force
+.PHONY: lint
+lint: musl proto proto-lint libs-lint auraed-lint auraescript-lint aer-lint ## Run all lints
```

Comment:
lint shouldn't need to depend on `proto` should it?

Reply:
I don't think `proto-lint` needs `proto`, but I'm pretty sure the lints for auraed, auraescript, & aer, require up-to-date proto files.

Reply:
i resolved this but then i realised: if they do then we should capture that in {auraed,auraescript,aer}-lint targets as dependencies.  make will only do it once (i think).

Reply:
I'll test

Reply:
It is true, and `proto` was already calling `proto-lint` which I realize I put in because I think the lint feedback should be immediate on regenerating the proto files